docs/REST/throttling.md

```diff
@@ -5,9 +5,13 @@ Configuration stores have limits on the requests that they may serve. Any reques
 Throttling is divided into different quota policies:
```

Comment:
This could be rephrased as follows or something along those lines to improve clarity.
`The requests to the configuration store are throttled based on the following quota policies.`